docs/topics/proc_configuring-jul-logging-vertx.adoc

```diff
@@ -0,0 +1,19 @@
+
+[#_configure_jul_logging_vertx]
+= Configuring logging for your {VertX} application using `java.util.logging`
+To configure debug logging for your {VertX} application using `java.util.logging`:
```

Comment:
Empty line missing between L3 and L4. I believe this might cause formatting issues in the resulting document.